api/src/s3/s3.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Req, Res, Param } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { S3Service } from './s3.service';
+
+@Controller('s3')
+export class S3Controller {
+  constructor(private s3service: S3Service) { }
+
+  @Get('/sign')
+  sign(@Req() req: Request, @Res() res: Response) {
+    return this.s3service.signIn(req, res);
+  }
+
+  @Get('/uploads/NestJsAdminBoilerplate/:caseId/:key')
+  fileRedirect(@Param('caseId') caseId: string, @Param('key') key: string, @Res() res: Response) {
```

Comment:
please rename

Reply:
it was renamed in next commits before your comment. Check, please.